### PR TITLE
Initial set up of experiment to show favorite sector loan on the feat…

### PR DESCRIPTION
**Pull request by mcstover**
…ured card.

### CODE REVIEW

**Comment by mcstover** [comment]
Waiting on final copy for EXP title and description.

File: src/components/LoansByCategory/FeaturedHeroLoanWrapper.vue [hunk 1]
```diff
@@ -65,12 +65,19 @@ export default {
...
+			type: String,
+			default: null
+		}
 	},
 	data() {
 		return {
 			experimentData: {},
-			featuredCategoryIds,
-			initialLoanCount,
+			featuredCategoryIds: [56],
```
Comment (mcstover) [inline src/components/LoansByCategory/FeaturedHeroLoanWrapper.vue]: These were never getting updated to have a value in the code below...So we'll just go with the hardcoded value for the default loan channel and the global limit.

File: src/components/LoansByCategory/FeaturedHeroLoanWrapper.vue [hunk 2]
```diff
@@ -80,72 +87,94 @@ export default {
...
-			this.loading = false;
+		if (this.favoriteSectorId !== null && this.featuredSectorExpVersion === 'shown') {
+			// set initial loan data so we ssr with a loan if possible
+			const loanArray = _get(allLoanData, 'lend.loans.values');
+			this.setInitialLoan(loanArray);
+			// display custom channel data
+			this.loanChannel = {
+				id: 999,
+				description: 'Some special text',
+				name: 'Some special title',
```
Comment (BoulderBrains) [inline src/components/LoansByCategory/FeaturedHeroLoanWrapper.vue]: :shipit:

File: src/graphql/query/featuredLoansData.graphql [hunk 3]
```diff
@@ -25,5 +26,21 @@ query featuredLoansData(
 				}
 			}
 		}
+		loans(
```
Comment (emuvente) [inline src/graphql/query/featuredLoansData.graphql]: Would it be much effort to do this as a separate query and select which loan query to call based on the experiment assignment?

Comment (mcstover) [reply src/graphql/query/featuredLoansData.graphql]: I think longer term I'd like to have a data provider we could alter params on to switch queries. But for this one, it was just more simple to run both to get to ensure this loan was rendered from the server using the prefetch.

**Pull request merged**